azure-pipelines-nightly.yml

```
@@ -558,11 +558,19 @@ jobs:
       IDF_PATH: "D:/a/1/s/esp-idf"
       PIP_CACHE_DIR: $(Pipeline.Workspace)/.pip
       TargetPlatform: "esp32"
+      IDF_TAG: "v5.2.3"
 
     steps:
       - checkout: self
-      - checkout: esp32_idf
-        fetchDepth: 1
+      - task: PowerShell@2
+        displayName: Checkout IDF 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)